types/open-location-code/.eslintrc.json

```diff
@@ -0,0 +1,6 @@
+{
+    "rules": {
+        // disable the rule for all files
+        "@typescript-eslint/explicit-member-accessibility": "off"
```

Comment:
why this exclusion? There are imo 2 problems in source code, both can be fixed or commented inline (public is default visibility)

Reply:
Thanks for your review.
I certainly don't need public, so I deleted it.

When I pushed the fix, an "Incorrect default export" error started appearing in CI. Do you know how to resolve this error?

Adding the esModuleInterop setting to tsconfig did not resolve the error.

I was able to run `pnpm run test-all` without any errors in my local environment.

Reply:
When I stopped `export default` and changed it to `export =`, the error no longer occurred.

Reply:
Is there a way to reproduce the same tests as CI in my local environment?